testsuite/meta/features/string_char.dat

```diff
@@ -0,0 +1,12 @@
+<<_SHOGUN_SERIALIZABLE_ASCII_FILE_V_00_>>
+array Vector<SGSerializable*> 2 ({Serializable int32 [
+value int32 12
+]}{Serializable int32 [
```

Comment:
no need to add this here ... no content :)
Currently we only serialize float32/64 scalar/vector/matrix.
We can change that of course, but it needs some more headachy changes ... and no need to do that before we actually have an algorithm exposed that actually produces string data

Reply:
OK, no problem. I will remove the data file.